device/juniper/x86_64-juniper_qfx5200-r0/pmon_daemon_control.json

```diff
@@ -1,3 +1,4 @@
 {
-  "skip_thermalctld": true
+  "skip_thermalctld": true,
+  "skip_pcied": true
```

Comment:
please add pcie.yml to enable this.

Reply:
@lguohan I'm not sure if I follow what you said. Are you asking to add 'device/juniper/x86_64-juniper_qfx5200-r0/pcie.yml' to skip the pcied?  With the content similar to the one below

{  ====> pcie.yml
    "skip_pcied": true
}
 
I couldn't see any similar files, instead I can see pmon_daemon_control.json

device/barefoot/x86_64-accton_wedge100bf_32x-r0/pmon_daemon_control.json:    "skip_pcied": false,
device/barefoot/x86_64-accton_wedge100bf_65x-r0/pmon_daemon_control.json:    "skip_pcied": true,
device/barefoot/x86_64-accton_as9516_32d-r0/pmon_daemon_control.json:    "skip_pcied": false,

Reply:
@lguohan Please share your thoughts on this.

Reply:
@lguohan I'm waiting for your input on this. Could you please clarify your comment?

Reply:
@sujinmkang @jleveque , do we have porting guide for supporting pcie.yml?

Reply:
No. @sujinmkang: Can you please add instructions for adding support for pcied to the [porting guide](https://github.com/Azure/SONiC/wiki/Porting-Guide)?

Reply:
@lguohan @jleveque @sujinmkang Meanwhile the documentation is getting updated, could you quickly tell me what needs to be done? I will rebase the patches and update the pull request.

Reply:
@lguohan Porting guide is not updated. Could you take this commit? Once the instructions are available for pcie.yml, I will submit another patch for moving to it.

Reply:
@ciju-juniper please find the updated porting guide.

Reply:
@sujinmkang How to disable pcied? Could you add that details also to porting guide? Please let me know how can it be disabled so that I can move ahead with this PR.

Reply:
@lguohan @jleveque @sujinmkang  Can you please clarify how to disable pcied? Is it just a matter of having an empty pcie.yaml or something else to be done specifically?

Reply:
Without the pcie.yaml, pcied will be disabled. But you can also add "skip_pcied": true, at pmon_daemon_control.json under platform directory.

Reply:
@sujinmkang That's exactly what I did initially. Please see the initial comment. I've already got pmon_daemon_control.json  Then @lguohan mentioned that it has changed.

@lguohan Could you please clarify what needs to be done now? I've already got what @sujinmkang  is recommending.

Reply:
@ciju-juniper is there any reason you don't want to add the pcie.yaml file to enable the pcied? You can easily generate the pcie.yaml file just using `pcieutil pcie-generate`. I know some platform vendors disabled the pcied on simulator platform not all the platform.

Reply:
@sujinmkang Yes. At this time we can't do the testing for it. More importantly I need to get this commit urgently as there are some customer requirements and the changes are there in this commit.

I shall enable pcied in the subsequent commits in a short while.

Reply:
Ok, please let me know when PR with pcie.yaml is available for your platforms.

Reply:
@sujinmkang Sure.. I will have the PR asap.
@lguohan Could you please merge this PR?

Reply:
@ciju-juniper can you just remove the skip_pcied part since we don't need that?

Reply:
@sujinmkang I've updated the patch set by removing 'skip_pcied' for both the platforms.